### DISCUSSION TITLE
allow for iOS 17 device debugging without editing sudoers

### DISCUSSION THREAD

**Discussion by nlash-livly** [wojciech-kulik/xcodebuild.nvim#138] [Ideas]
If it's possible (even with promping for password), is there a way to expose running on devices without having to edit sudoers? My work prevents us from editing sudoers.

**Comment by wojciech-kulik** [comment]
Have you managed to solve it?

**Comment by nlash-livly** [comment]
Yes, I was able to obtain access to sudoers. Would be ideal not to have to do this, but I know how much of a struggle Xcode 15.x changes have been.
________________________________

**Reply by wojciech-kulik** [reply]
Great to hear! Yeap, they complicate things more and more for 3rd party tools...
